content/06.methods.md

```diff
@@ -174,7 +174,7 @@ We used GISTIC [@doi:10.1186/gb-2011-12-4-r41] v.2.0.23 on the CNVkit and the co
 ##### Consensus CNV Calling
 
 For each caller and sample, CNVs were called based on consensus among Control-FREEC [@doi:10/ckt4vz; @doi:10/c6bcps], CNVkit [@doi:10.1371/journal.pcbi.1004873], and Manta [@doi:10/gf3ggb].
-CNVs called significant by Control-FREEC (p-value < 0.01) and Manta calls that passed all filters [@url:https://github.com/Illumina/manta/blob/75b5c38d4fcd2f6961197b28a41eb61856f2d976/docs/userGuide/README.md#vcf-format-fields] were included in consensus calling.
+CNVs called significant by Control-FREEC (p-value < 0.01) and PASS Manta calls were included in consensus calling.
```

Comment:
I think we can remove the URL, but the previous wording was more clear in my opinion

```suggestion
CNVs called significant by Control-FREEC (p-value < 0.01) and Manta calls that passed all filters were included in consensus calling.
```